threedi_modelchecker/tests/test_checks_other.py

```diff
@@ -366,3 +367,26 @@ def test_potential_breach_interdistance_other_channel(session):
     )
     invalid = check.get_invalid(session)
     assert len(invalid) == 0
+
+
+@pytest.mark.parametrize(
+    "storage_area,time_step,expected_result",
```

Comment:
Can you also add a test for `storage_area = None` and pump capacity `capacity = 0`?